src/features/deposit/components/DepositFormNetwork.tsx

```diff
@@ -0,0 +1,71 @@
+import { useForm } from "react-hook-form"
+
+import { Button, Spinner, Text } from "@radix-ui/themes"
+import { Form } from "src/components/Form"
+import { Select } from "src/components/Select/Select"
+import type {
+  BlockchainEnum,
+  DepositFormNetworkProps,
+  DepositFormNetworkValues,
+} from "../../../types/deposit"
+
+export const DepositFormNetwork = ({
+  blockchains,
+  onSubmit,
+}: DepositFormNetworkProps) => {
+  const {
+    handleSubmit,
+    register,
+    setValue,
+    watch,
+    formState: { errors },
+  } = useForm<DepositFormNetworkValues>({ reValidateMode: "onSubmit" })
+
+  return (
+    <div className="w-full md:max-w-[472px] rounded-[1rem] p-5 shadow-paper bg-white dark:shadow-paper-dark dark:bg-black-800">
```

Comment:
We're mixing Tailwind with plain css again. Maybe let's stick to one so it will be easier to migrate in the future?